util/abs/provision_pe_xl_nodes.rb

```diff
@@ -37,6 +39,7 @@
 DEFAULTS = <<~DEFAULTS
 
   The following defaults values are used if the options are not specified:
+  * REF_ARCH (--ref_arch): #{DEFAULT_REF_ARCH}
```

Comment:
Not sure if it is necessary to include the short forms here... if so we should add `-a`; if not `-i` and `-o` should be removed.

Reply:
I have updated to include the `-a` short form.